packages/react-native-bpk-component-text-input/src/__snapshots__/BpkTextInput-RTL-test.js.snap

```diff
@@ -2638,7 +2793,7 @@ exports[`RTL BpkTextInput light mode should support theming 1`] = `
       style={
         Object {
           "color": "rgba(178, 178, 191, 1)",
-          "fontFamily": "System",
+          "fontFamily": undefined,
```

Comment:
Is this expected? Shouldn't it say 'relative'

Reply:
It should not be set because this test is only providing the required props, let me double check if the end result is what we expect.

Reply:
That was wrong, good catch! Because that text was only providing the required theme attribute the `themeAttributes` variable was not null, but `textFontFamily` was. It's fixed now @ojcurtis